sample-app.js

```diff
@@ -196,6 +196,36 @@ async function start() {
     }
   });
 
+  server.route({
+    method: 'GET',
+    path: '/creators/top_holders_and_transactions',
+    handler: async function(request, h) {
+      if (!access_token) {
+        return h.response("Application Not Registered With Rally").code(401);
+      }
+      console.log(`params = ${JSON.stringify(request.query)}`);
+      console.log('token: ' + access_token);
+
+      const rnbUserId = request.query.rnbUserId;
+      const symbol = request.query.symbol;
+      const timePeriod = request.query.timePeriod;
+
+      console.log('Calling GET /creators/top_holders_and_transactions');
+      const rally_response = await httpGet(
```

Comment:
We should make the API a POST rather than a GET and pass the arguments as body parameters.
Since we're using authentication, the GET endpoint can't easily be hit from the browser address-line

Reply:
so the idea is to change this into a post? it's not changing any data though, is it?

Reply:
Discussed it with Alain, it should be GET.